mailet/api/src/main/java/org/apache/mailet/Mail.java

```diff
@@ -98,7 +99,9 @@
      * @throws MessagingException when an error occurs while retrieving the message
      */
     MimeMessage getMessage() throws MessagingException;
-    
+
+    Message getParsedMimeMessage() throws MessagingException;
```

Comment:
@chibenwa I know you said you wanted a `Optional<Message>` but I could not figure out why. When a message would not have a mime content ?